### PR TITLE
Migration/stem

### PR DESCRIPTION
**Pull request by rvilarl**
Another quick win!

### CODE REVIEW

File: src/stem.ts [hunk 1]
```diff
@@ -7,116 +7,156 @@
...
 
 // To enable logging for this class. Set `Vex.Flow.Stem.DEBUG` to `true`.
-function L(...args) {
+function L(
+  // eslint-disable-next-line
+  ...args: any[]) {
   if (Stem.DEBUG) Vex.L('Vex.Flow.Stem', args);
 }
 
+export interface StemStruct {
```
Comment (0xfe) [inline src/stem.ts:19]: Let's call all options parameters `*Options` so 'StemOptions' here. (`NoteStruct` is a legacy wart to clean up.)

**Comment by 0xfe** [comment]
Looks good. Just rename `StemStruct` please, and I can merge.

File: src/stem.ts (hunk 1 above)
Comment (rvilarl) [reply src/stem.ts:19]: ok

**Comment by rvilarl** [comment]
Ready.

**Comment by 0xfe** [comment]
Looks good.

**Pull request merged**